inc/class-thumbnails.php

```diff
@@ -170,7 +173,16 @@ public function get_data( $post, $url ) {
 		$thumbnail_path = '';
 		$thumbnail_url = '';
 		
-		if ( strpos( $url, 'vimeo.com' ) !== false ) {
+        if ( strpos( $url, 'slideshare.net' ) !== false ) {
```

Comment:
It seems that the indentation is corrupted here. 🙂

Reply:
Oh right, you use tabs and I had my editor set up with spaces. I'll fix it now.